rudder-webapp/SOURCES/rudder-init

```diff
@@ -179,12 +179,13 @@ cpt=0
 REQUIRE_NET=""
 for i in ${ALLOWEDNETWORK[*]}
 do
-  if [ $cpt -eq 0 ]
+# comma between element of array
+  if [ $cpt -ne 0 ]
   then
-    NET="$i"
-  else
-    NET="$NET\ndirectiveVariable: ALLOWEDNETWORK[$cpt]:$i"
+    NET="$NET,"
   fi
+
+  NET="$NET\n {\"name\":\"root init allowed network $cpt\",\"inet\":\"$i\"}"
```

Comment:
we use ${NET} in string to make sure we never have conflicts in variable interpretation

Reply:
it was not the case, neither for `$i` and `$cpt`, so I thought it was intentionnal. Should I change all of them?